### ISSUE TITLE
Anaconda xgboost status remains non Windows.

### ISSUE THREAD

**Issue by JimSEOW** [dmlc/xgboost#4272]
As the issue with Windows 10 installation has been resolved recently #4197 
The deepchem community needs Anaconda XGBoost Win64 to attempt installing DeepChem to Windows 10 without WSL so we could take advantage of tensorflow-GPU.

![image](https://user-images.githubusercontent.com/8244633/54520135-471cb180-4968-11e9-82d3-bfd02bdd351c.png)

**Comment by hcho3** [comment]
@JimSEOW Can you use the binary wheels from the Python Package index? https://pypi.org/project/xgboost/#files

We maintainers do not maintain the Anaconda distribution; we only manage the PyPI package. Can you contact the maintainer of the Anaconda XGBoost package directly?

**Comment by hcho3** [comment]
@JimSEOW Any updates?
